### PR TITLE
Add Lambda Handler Function and Dockerfile for Alerts Service

### PR DESCRIPTION
**Pull request by ebradley12**
This pull request addresses tickets #105 and #106

**Description**
This pull request introduces the Lambda handler function and the Dockerfile necessary for the Alerts Service in the Bandcamp Tracker project. The Alerts Service generates artist and genre-based alerts for subscribers based on trends and sends them via email. 

*Docker image still needs to be pushed to ECR once alerts.py script has been completed.*

### CODE REVIEW

File: alerts/lambda_handler.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+"""
+This script is designed to be the entry point for an AWS Lambda function.
+The purpose of the Lambda function is to to monitor sales data, detect changes
+in top artists and genres, and send alert emails to subscribers using AWS SES.
+"""
```
Comment (emily-curtis) [inline alerts/lambda_handler.py:5]: Nice introduction to the script!

File: alerts/lambda_handler.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    Terminal logs configuration.
+    """
+    logging.basicConfig(
+        format="%(asctime)s - %(levelname)s - %(message)s",
+        level=logging.INFO,
+    )
+
+
+def lambda_handler(event, context):
+    """
```
Comment (emily-curtis) [inline alerts/lambda_handler.py:21]: could we add some typehints?

**Review by emily-curtis — APPROVED** [review]
Looks great, nothing to fault!

File: alerts/lambda_handler.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+"""
+This script is designed to be the entry point for an AWS Lambda function.
+The purpose of the Lambda function is to to monitor sales data, detect changes
+in top artists and genres, and send alert emails to subscribers using AWS SES.
+"""
+import logging
+from alerts import main
+
+
+def configure_logging():
```
Comment (bentrzcinski) [inline alerts/lambda_handler.py:10]: Could add '-> None' type hints to functions with no output?

File: alerts/lambda_handler.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+        logging.info("Alerts processing completed successfully.")
+        return {
+            "statusCode": 200,
+            "body": "Alerts sent successfully."
+        }
+    except Exception as e:
+        logging.error(f"Error during alerts processing: {e}")
+        return {
+            "statusCode": 500,
```
Comment (bentrzcinski) [inline alerts/lambda_handler.py:36]: Is the only error code we could ever see a 500?

File: reports/queries.py [hunk 5]
```diff
@@ -86,6 +86,7 @@ def query_top_artists(cursor, date: str) -> list:
         JOIN release AS r ON s.release_id = r.release_id
         JOIN artist AS a ON r.artist_id = a.artist_id
         WHERE DATE(s.sale_date) = %s
+        AND a.artist_name != 'Various Artists'
```
Comment (bentrzcinski) [inline reports/queries.py:89]: This is great and I'm going to steal it

**Review by bentrzcinski — COMMENTED** [review]
Nice code as per usual

File: reports/queries.py (hunk 5 above)
Comment (ebradley12) [reply reports/queries.py:89]: Theif!

File: alerts/lambda_handler.py (hunk 4 above)
Comment (ebradley12) [reply alerts/lambda_handler.py:36]: This is just a generic error code. Theres lots of error handling covered within the main code so I don't think we need to be too specific here

File: alerts/lambda_handler.py [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    """
+    Terminal logs configuration.
+    """
+    logging.basicConfig(
+        format="%(asctime)s - %(levelname)s - %(message)s",
+        level=logging.INFO,
+    )
+
+
+def lambda_handler(event: dict, context=None):
```
Comment (bentrzcinski) [inline alerts/lambda_handler.py:20]: And a '-> None' here?

**Review by bentrzcinski — CHANGES_REQUESTED** [review]

**Review by bentrzcinski — APPROVED** [review]
Lovely stuff

**Pull request merged**